File: python-services/entities/correntista.py
class Correntista:
    def __init__(self, id=None, nome="", cpf="", email="", telefone="", data_nascimento=None, endereco=""):
        self.id = id
        self.nome = nome
        self.cpf = cpf
        self.email = email
        self.telefone = telefone
        self.data_nascimento = data_nascimento
        self.endereco = endereco
    
    def to_dict(self):
        return {
            'id': self.id,
            'nome': self.nome,
            'cpf': self.cpf,
            'email': self.email,
            'telefone': self.telefone,
            'data_nascimento': str(self.data_nascimento) if self.data_nascimento else None,
            'endereco': self.endereco
        }

File: backend/src/main/java/entity/Correntista.java
package com.aurum.bank.entity;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;

@Entity
@Table(name = "correntista")
@Data
public class Correntista {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_correntista")
    private Long id;
    
    @Column(nullable = false, length = 100)
    private String nome;
    
    @Column(unique = true, nullable = false, length = 11)
    private String cpf;
    
    @Column(length = 100)
    private String email;
    
    @Column(length = 20)
    private String telefone;
    
    @Column(name = "data_nascimento")
    private java.sql.Date dataNascimento;
    
    @Column(columnDefinition = "TEXT")
    private String endereco;
    
    @Column(name = "data_criacao")
    private LocalDateTime dataCriacao = LocalDateTime.now();
}